translate/v3/batch_translate_text_with_glossary_and_model_test.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v3
+
+// TODO: uncomment this test once the AutoML model is set in the testing projects.
```

Comment:
What's the `gcloud` command to do this? I'll run it.

Reply:
Unfortunately, it's not a single command. It's following [these instructions](https://cloud.google.com/translate/automl/docs/before-you-begin). It's a lot of manual labor and training the model takes several hours that will have to be done on all testing projects.

I was writing a script to do it automatically, but it I also got into [this bug](https://buganizer.corp.google.com/issues/143495873) when importing the data using the REST API. I'll give it another try with the Python client library.

In any case, we wanted to publish the samples for the GA of translation v3, so I was thinking on merging this PR and having a follow up PR once the AutoML models are set up since that will take more time.